package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@swapr/frontend",
-  "version": "1.0.0-beta.22.3",
+  "version": "1.0.0-beta.23.0",
   "private": true,
   "description": "DXswap Decentralized Application",
   "license": "GPL-3.0-or-later",
   "scripts": {
     "start": "yarn codegen:socket && yarn codegen:graphql && react-app-rewired start",
-    "build": "yarn codegen:wallet-connect-domain-verification && yarn codegen:socket && yarn codegen:graphql && cross-env GENERATE_SOURCEMAP=false react-app-rewired build",
+    "build": "git config --global --add safe.directory /workspace && yarn codegen:wallet-connect-domain-verification && cross-env GENERATE_SOURCEMAP=false react-app-rewired build",
+    "build:simple": "git config --global --add safe.directory /workspace && cross-env GENERATE_SOURCEMAP=false react-app-rewired build",
```

Comment:
can you explain a bit this different builds? 

Maybe would be good to have that on the readme